TravelBug/src/WebPages/SingleVideo/SingleVideo.jsx

```diff
@@ -0,0 +1,52 @@
+import { useParams } from "react-router-dom";
+import { VideoCard } from "../../Components";
+import { useData } from "../../Contexts/Datacontext";
+import "./SingleVideo.css";
+
+const SingleVideo = () => {
+  const { state } = useData();
+  const { videos } = state;
+  const { vidId } = useParams();
+  const video = videos?.find((element) => element._id === vidId) || {};
+  const otherVideoList = videos?.filter((element) => element._id != vidId);
+  console.log("vid ", vidId);
+  console.log("videos", videos);
+  console.log("video here ", video);
```

Comment:
Just a reminder to remove these before submission.

Reply:
Done